subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/RepositoryHandler.java

```diff
@@ -119,7 +119,9 @@
      *
      * @param action a configuration action
      * @return the added repository
+     * @deprecated JCenter will soon be <a href="https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter">shut down</a>. Use {@link #mavenCentral()} instead.
```

Comment:
@donat 
@jjohannes 
Just my opinion and not very important but such a depecration warning could and will be read in the far future, for example in 2022 when a developer copy paste an unmaintained github.com repository using jcenter(). 
He will read that it *JCenter will soon be shut down* despite being already shut down since ~6 months (or n years). 

Instead he could immediately understand by rephrasing the deprecation message to 
"JCenter will be <a href="https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter">shut down</a> the 01/05/2021. Use {@link #mavenCentral()} instead.